integration-testing/resource_mongodbatlas_encryption_at_rest_test.go

```diff
@@ -1,12 +1,12 @@
-// +build integration
```

Comment:
[q] I've been setting up the repo thru the day and going with docs code, is the plan to stop taging integration tests? it seems some are using it and some are not, can we decide on this and updates docs accordingly (some of the docs on how to test reference the use of tags)?